panel/models/echarts.ts

```diff
@@ -46,7 +46,7 @@ export class EChartsView extends HTMLBoxView {
     this.connect(this.model.properties.data.change, () => this._plot())
     const {width, height, renderer, theme, event_config, js_events} = this.model.properties
     this.on_change([width, height], () => this._resize())
-    this.on_change([theme, renderer], () => this.render())
+    this.on_change([theme, renderer], () => {this.render(), this._chart.resize()})
```

Comment:
Without this change the `echarts` plots will not be very wide after a dynamic change of the theme.